app/views/pages/home.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="home_page">
+  <%= image_tag 'theme2.jpg' %>
+  <div class="app_description d-flex justify-content-center">
+    <div style="width: 18rem;">
+      <%# <%= current_user.name %>
```

Comment:
Let's have a clean codebase, by removing unused/unnecessary comments. It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))